test/coreneuron/test_pointer.py

```diff
@@ -269,32 +293,22 @@ def run(tstop):
         assert spikes_std == spikes
     coreneuron.enable = False
 
-    # standard to compare with checkpoint series
-    tpnts = [5.0, 10.0]
-    for perm in [0, 1]:
-        print("\n\ncell_permute ", perm)
-        common = "-d coredat --voltage 1000 --verbose 0 --cell-permute %d" % (perm,)
-        # standard full run
-        runcn(common + " --tstop %g" % float(tpnts[-1]) + " -o coredat")
-        # sequence of checkpoints
-        for i, tpnt in enumerate(tpnts):
-            tend = tpnt
-            restore = " --restore coredat/chkpnt%d" % (i,) if i > 0 else ""
-            checkpoint = " --checkpoint coredat/chkpnt%d" % (i + 1,)
-            outpath = " -o coredat/chkpnt%d" % (i + 1,)
-            runcn(
-                common + " --tstop %g" % (float(tend),) + outpath + restore + checkpoint
-            )
-
-        # compare spikes
-        cmp_spks(
-            spikes_std, "coredat", ["coredat/chkpnt%d" % (i,) for i in range(1, 3)]
-        )
-
+    # Delete model before launching the CoreNEURON simulation offline
     m._callback_setup = None
     pc.gid_clear()
     del m
 
+    # Shrink memory pools of mechanisms
+    h.CVode().poolshrink()
```

Comment:
This does not actually shrink the pool (see `void nrn_poolshrink(int shrink)`), we'd need to supply `1`. 
I think we should document this feature in this PR.

Reply:
Done. Thanks for pointing that out :+1: